ui/core/src/test/autosuggest/AutoSuggest.test.tsx

```diff
@@ -199,6 +201,70 @@ describe("AutoSuggest", () => {
     document.body.removeChild(outerNode);
   });
 
+  it("should support renderInputComponent prop", async () => {
+    const outerNode = document.createElement("div");

```

Comment:
Why a custom `outerNode` is needed to render an AutoSuggest component?

Reply:
The AutoSuggest component does not open properly in the unit tests without the outerNode.